VAMobile/src/store/api/demo/store.ts

```diff
@@ -224,6 +228,9 @@ const transformGetCall = (endpoint: string, params: Params): DemoApiReturns => {
     case '/v1/health/immunizations': {
       return getVaccineList(store, params, endpoint)
     }
+    case '/v1/health/allergy-intolerances': {
```

Comment:
Resolve discrepancy between this and the endpoint in use using the `v0` namespace

Reply:
gonna address this with labs & tests